src/test/java/teammates/ui/webapi/GetFeedbackResponseCommentActionTest.java

```diff
@@ -39,7 +39,7 @@ protected String getRequestMethod() {
     }
 
     @Override
-    protected void prepareTestData() {
+    protected void setUp() {
```

Comment:
I think the old test case's setup method name needs to be kept as `prepareTestData()`, since it overrides the method in `BaseActionTest`.  perhaps this was a mistake? I think this will fix the failing component test